os/se/ameba/security_ameba_wrapper_tz.c

```diff
@@ -0,0 +1,763 @@
+/**************************************************************************************************
+*
+* Copyright 2020 Samsung Electronics All Rights Reserved.
+*
+* Licensed under the Apache License, Version 2.0 (the "License");
+* you may not use this file except in compliance with the License.
+* You may obtain a copy of the License at
+*
+* http://www.apache.org/licenses/LICENSE-2.0
+*
+* Unless required by applicable law or agreed to in writing,
+* software distributed under the License is distributed on an
+* "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND,
+* either express or implied. See the License for the specific
+* language governing permissions and limitations under the License.
+*
+***************************************************************************************************/
+
+#include <tinyara/config.h>
+#include <semaphore.h>
+#include <tinyara/seclink_drv.h>
+#include <tinyara/security_hal.h>
+#include <tinyara/kmalloc.h>
+
+#include <stdint.h>
+#include <stdio.h>
+#include <stdlib.h>
+#include <string.h>
+
+#define AWRAP_TAG "[AMEBA_WRAPPER]"
+#define AWRAP_ENTER                                                                 \
+	do {                                                                             \
+		sedbg(AWRAP_TAG"[INFO] %s %s:%d\n", __FUNCTION__, __FILE__, __LINE__);       \
```

Comment:
Actually these are refer from os/se/sss/security_hal_wrapper.c, but there some different. 
for os/se/sss/security_hal_wrapper.c it have removed **FUNCTION**.
Should i keep this or just remove it? Since you say it is not helpful.